### DISCUSSION TITLE
How to add additional language

### DISCUSSION THREAD

**Discussion by timothybohte** [khoait/DCE.PCF#128] [Q&A]
Great that you added multiple language support!
In the documentation you are referring to creating a resx-file for PolyLookup.
Could you pleae share the XML-format and an example of this file?

**Comment by khoait** [comment]
hi @timothybohte thanks for your interest in the component! you can add your language by passing a URL to a .resx web resource to the `Language Pack Path` property.

you can use the English language pack here [PolyLookup.1033.resx](https://github.com/khoait/DCE.PCF/blob/main/PolyLookupComponent/PolyLookup/strings/PolyLookup.1033.resx) as the template to create a new resx web resource in your environment, then update the value in your language. 

you can use this VS Code extension to help you edit the resx web resource in a user interface https://marketplace.visualstudio.com/items?itemName=DominicVonk.vscode-resx-editor

For example, the resx web resource looks like this

<img alt="resx web resource" src="https://github.com/user-attachments/assets/40799980-dc7f-4225-bbb6-b8085d1ca973" width="400" />

the value passed to the `Language Pack Path` can be relative and won't need the environment part. it looks like this:
```
/WebResources/storm_/resx/PolyLookup.1031.resx
```
